static/js/util/sorting.js

```diff
@@ -15,12 +17,17 @@ export function resumeOrder(entries: WorkHistoryEntry[], dateFieldName: string):
     }
     return 0;
   };
-  return entries.sort(sortFunc);
+  return clone.sort(sortFunc);
```

Comment:
this can be abstracted further, eg:

```javascript
dateCompare (a, b) {
  if ( a.isBefore(b) ) {
    return 1;
  }
  if ( a.isAfter(b) ) {
    return -1;
  }
  return 0;
}

export function resumeOrder(entries: [number, Object][], dateFieldName: string): any {
  let clone = _.clone(entries);
  let sortFunc = ([, a], [, b]) => {
    return dateCompare(moment(a[dateFieldName]), moment(b[dateFieldName]));
  }
  return clone.sort(sortFunc);
}
```